lib/iris/__init__.py

```diff
@@ -44,6 +44,8 @@
         standard library function :func:`os.path.expanduser` and
         module :mod:`fnmatch` for more details.
 
+        If supplying a URL, only OPeNDAP Data Sources are supported.
```

Comment:
@lbdreyer Great to highlight this to the user :100: 

Perhaps we could also add a `.. warning::` directive in the above main body of the doc-string to ensure this information can't be missed in the rendered `sphinx` docs?